StableIPFSLoader.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="fragment" content="!">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Decentralized Prediction Markets | Augur</title>
+  <meta name="description"
+        content="Augur is a decentralized oracle and prediction market protocol built on the Ethereum blockchain. It allows you to forecast events and be rewarded for predicting them correctly.">
+  <script src="https://unpkg.com/ipfs/dist/index.min.js"></script>
+  <script>
+      (async function() {
+          const node = await window.Ipfs.create();
+          const pageData = await fetch(window.location.href);
+
+          if (!pageData.ok) {
+              return console.error(pageData.statusText);
+          } else {
+              await node.add(await pageData.text(), (error, response) => {
```

Comment:
Do we know that this produces the same hash as is being loaded right now?

Reply:
Oh also when in await mode there's no need to pass the callback in as it waits.

```
const response = await node.add(pageData.text());
```

Where the error will get thrown.

Reply:
> Do we know that this produces the same hash as is being loaded right now?

That's the hash from the static page (StableIPFSLoader.html)